doc/DEPLOYMENT.md

```diff
@@ -107,87 +109,36 @@ Now, setup an IAM role using the *terraform-iam* module with just enough permiss
 	- Update "repo_name" to be "deployment-name-secrets"
 	- Update the user ARN to reflect your user
 - `terraform init`
-- `awsudo arn:aws:iam::162808325377:role/<deployment-name>-secrets-setup terraform apply -var-file=roles.tfvars`
+- `awsudo arn:aws:iam::<account-id>:role/<deployment-name>-terraform-architect terraform apply -var-file=roles.tfvars`
```

Comment:
did you actually get rid of the secrets-setup role and increase terraform-architect's perms or was this just planned?

Reply:
doesn't actually matter, the secrets role is going away

Reply:
but does it work? does terraform-architect have the perms to do this step?

Reply:
as of now yes.  it does work